src/app/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -32,10 +32,11 @@ import type {
   PDFViewerOptions,
   ZoomScale
 } from './typings';
-import { PDFSinglePageViewer } from 'pdfjs-dist/web/pdf_viewer';
+import { PDFSinglePageViewer } from 'pdfjs-dist/web/pdf_viewer.mjs';
+import { GlobalWorkerOptions, VerbosityLevel, getDocument } from 'pdfjs-dist';
```

Comment:
Why do you import "pdfjs-dist" again when it is already imported in line 20?

Reply:
Because its a deep import and its not available from the top level

Reply:
I might be a little rusty with my angular, but I always thought that this

```
import * as PDFJS from 'pdfjs-dist';

console.log(PDFJS.VerbosityLevel.INFOS)
```

is functionally the same as this.

```
import { VerbosityLevel } from 'pdfjs-dist';

console.log(VerbosityLevel.INFOS)
```

Sure - doing a deep import is more treeshakeable, and is thus preferable, but what you are telling me (or at least that is what I am understanding) is, that there is a difference and both imports are necessary. Did I get that right?

Reply:
Is this a show stopper?

Reply:
No because  it has to be deep